automation/dependDockerfile/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM quay.io/openshift/origin-jenkins-agent-base:v4.0.0
+# Jenkins slave requirements
+RUN yum makecache \
+ && yum erase -y java-1.8.0-openjdk-headless \
+ && yum -y install openssl java-1.8.0-openjdk \
+ && pushd /etc/pki/ca-trust/source/anchors \
+ && curl -s -k -O https://password.corp.redhat.com/RH-IT-Root-CA.crt \
+ && curl -s -k -O https://password.corp.redhat.com/legacy.crt \
+ && curl -s -k -O https://engineering.redhat.com/Eng-CA.crt \
+ && update-ca-trust extract \
+ && openssl x509 -in RH-IT-Root-CA.crt -out RH-IT-Root-CA.der -outform der \
+ && openssl x509 -in legacy.crt -out legacy.der -outform der \
+ && openssl x509 -in Eng-CA.crt -out Eng-CA.der -outform der \
+ && export JAVA_HOME="/usr/lib/jvm" \
+ && export PATH="${PATH}:${JAVA_HOME}/jre/bin" \
+ && keytool -importcert -noprompt -alias RH-IT-Root-CA -keystore $JAVA_HOME/jre/lib/security/cacerts -storepass changeit -file RH-IT-Root-CA.der \
+ && keytool -importcert -noprompt -alias legacy -keystore $JAVA_HOME/jre/lib/security/cacerts -storepass changeit -file legacy.der \
+ && keytool -importcert -noprompt -alias Eng-CA -keystore $JAVA_HOME/jre/lib/security/cacerts -storepass changeit -file Eng-CA.der
+ENTRYPOINT ["/usr/local/bin/run-jnlp-client"]
+# General requirements                                                          
+RUN yum install -y epel-release \
+ && yum install -y gcc python-devel \
+ && yum install -y python-pip \
+ && yum install -y git \
+ && git clone --depth 1 https://github.com/RedHatQE/CodeQuality
```

Comment:
It seems to be used only in line 44, and it seems to be something that should be part of testing and not the image?

Reply:
I thought to download the examples from the repo to the image in order to save some time in the testing process, but now I think you right, I'll remove this line from the image